src/walletConnect/index.tsx

```diff
@@ -752,21 +752,9 @@ export async function onSessionRequest(event: SignClientTypes.EventArguments['se
       },
     };
 
-    const { requests: pendingRequests } = store.getState().requests;
```

Comment:
for the WC v2 flow, we used to directly update the `redux/requests` redux state as well as localstorage - now we use the new requests store